sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -393,6 +393,45 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * This function goes through the input once to determine the input schema. To avoid going
    * through the entire data once, specify the schema explicitly using [[schema]].
    *
+   * You can set the following CSV-specific options to deal with CSV files:
+   * <li>`sep` or `delimiter` (default `,`): sets the single character as a delimiter for each
+   * field and value.</li>
+   * <li>`encoding` or `charset` (default `UTF-8`): decodes the CSV files by the given encoding
```

Comment:
Aliases were added for `codec`, `charset` and `delimiter` as `compression`, `charset` and `sep`.
I remember It was decided that`codec` is not documented but supported for backword compatibility because it looks not a good name.
As I thought both `sep` and `delimiter` or `encoding` or `charset` are good names, I just added but I am happy to get rid of them if any thinks not.